package.json

```diff
@@ -5,8 +5,10 @@
   "main": "index.js",
   "scripts": {
     "start": "cd backend && npm run create-schema && cd .. && ./start.sh",
-    "postinstall": "cd backend && npm install && cd ../frontend && npm install && npm run build && cd ..",
-    "lint": "eslint --max-warnings 0 './backend/**/*.ts' './frontend/src/**/*.js' --ignore-pattern '/backend/node_modules/*' --ignore-pattern '/frontend/node_modules/*'"
+    "lint": "eslint --max-warnings 0 './backend/**/*.ts' './frontend/src/**/*.js' --ignore-pattern '/backend/node_modules/*' --ignore-pattern '/frontend/node_modules/*'",
+    "start:e2e": "concurrently \"cd backend && npm start\" \"wait-on http://localhost:3000/games && cd frontend && npm run start:e2e\"",
```

Comment:
do we need to specifically wait on "/games" endpoint, or is "/" enough?

Reply:
wait-on waits for 2xx on GET and for "/" we give 404 only right now.

Reply:
a pity we cannot possibly make comments on this in a .json file. I'd think / should return a 200 in order to have a healthcheck implementable on it. in it's simplest form `{alive: true}` should be a sufficient response. Let's leave it for the future